.github/workflows/iroha2-dev-base_coverage.yml

```diff
@@ -1,25 +1,25 @@
-name: I2::Dev::BaseCoverage
+# name: I2::Dev::BaseCoverage
```

Comment:
Assuming this PR resolves #2781, this workflow should be removed instead of commented out. Even if it does not work we can revert

Reply:
All right, I implied to comment it temporarily due I am not sure that this thing will help.